recipes/acpype/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "acpype" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+    url: https://github.com/alanwilter/{{ name }}/archive/d99be8e17e3f999e06a401e5efdbe5f2fc9a2959.tar.gz
+    sha256: 43736873feaa791c76ee02a6eae6aba5fb661149a915d8772ec5a4a1f920473d
+build:
+    number: 0
+    script: python recipe/setup_conda.py install --single-version-externally-managed --record=record.txt
+    skip: True  # [win or osx]
```

This looks like it could be [`noarch python`](http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python)? It's find if Windows and macOS then fail due to missing dependencies as it will only be build on Linux:
```suggestion
    noarch: python

```

---

There also needs to be [entry points](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points) here to match with the [`console_scripts` in `setup.py`](https://github.com/alanwilter/acpype/blob/master/recipe/setup_conda.py#L25).